app/templates/datasets.hbs

```diff
@@ -54,14 +54,11 @@
               .csv
             </a>
 
-            {{#if model.raw_data.spatialMetaData}}
+            {{#unless model.raw_data.spatialMetaData}}
```

Comment:
This will make the spatial button show up only if there isn't spatial metadata

Reply:
So this particular piece of code is a bit confusing. What it is looking for the spatialMetadata about tabular data, not the spatialMetadata about spatial data. We can refactor it to name this something else, but if you test this on staging or in development you should see that it properly hides the shapefile download for tabular data and shows it for geospatial.

Reply:
```js
model.datasets.get('schemaname')
```